hw/ip/spi_host/dv/spi_host_sim_cfg.hjson

```diff
@@ -110,6 +110,11 @@
       uvm_test_seq: spi_host_status_stall_vseq
       run_opts: ["+en_scb=0"]
     }
+    {
+      name: spi_host_idlecsbactive
+      uvm_test_seq: spi_host_idlecsbactive_vseq
+      run_opts: ["+en_scb=0"]
```

Comment:
it seems like the test only gets to that FSM state but it doesn't check anything. Can you enable the scb? keeping scb high for next transaction shouldn't affect the data transfer (we may need to tune the scb or monitor a bit)

Reply:
Fixed and SB handles as usual.